configure-vscode.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
```

Comment:
_question_

Just curious, why is this file needed here if it's already in Agoric/agoric-sdk?

Reply:
The `.vscode/settings.json` file that this script produces only applies when it's at the root of the currently opened workspace, so running it from `agoric-sdk` wouldn't apply the settings to this repo unless they were opened together in the same workspace.

We could point to the `agoric-sdk` repo and ask people to copy that `.vscode/settings.json` into this workspace when working with this repo, but it's more convenient to just have it here and allows us to tweak things specific to this repo.

Reply:
https://code.visualstudio.com/docs/getstarted/settings